### DISCUSSION TITLE
Update coming to API alerts endpoint responses

### DISCUSSION THREAD

**Discussion by jah1007** [weather-gov/api#484] [General]
Wanted to alert everyone to SCN 21-33 (https://www.weather.gov/media/notification/pdf2/scn21-33_caphandler_v1.2_aac.pdf).  The upstream source of data for the /alerts and /alerts/active endpoints is being updated to align the CAP messages generated by the National Weather Service closer to the CAP standard. Please review the SCN and update applications accordingly.

**Comment by JJKraw** [comment]
If I read this correctly, this is only an issue if one is using the PIL parameter currently. Correct?

**Reply by jah1007** [reply]
That is correct.
